.github/workflows/merge-up.yml

```diff
@@ -6,12 +6,18 @@ on:
     - cron: '20 11 * * 6'
   workflow_dispatch:
 
+permissions: {}
+
 jobs:
   merge-up:
     name: Merge-up
     # Only run cron on the silverstripe account
     if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      actions: write
+      pull-requests: write
```

Comment:
`pull-requests: write` is not included in the list of expected permissions you provided: https://github.com/silverstripe/silverstripe-framework/issues/10510#issuecomment-2055406330

Can you please confirm if that's needed for this one?
We also discussed double checking whether raw `git` requires `contents: write` which I **think** was related to this one as well. Can you please confirm you have validated that?

Reply:
Yup looks like it's not required, will remove pull-requests: write